src/ploneintranet/workspace/workspacecontainer.py

```diff
@@ -15,3 +15,12 @@ class WorkspaceContainer(Container):
     A folder to contain WorkspaceFolders
     """
     grok.implements(IWorkspaceContainer)
+
+try:
```

Comment:
I think it's slightly more readable this way, because the import is only used in that small block that is in itself a bit of an afterthought. That said, we probably want a hard dependency on ploneintranet.attachments ultimately, so we would turn it into an unconditional import and maybe also do the "implements" differently.

Actually, it's probably time to change that now. I'll look into it.